glslc/src/file_compiler.h

```diff
@@ -57,24 +60,45 @@ class FileCompiler : public shaderc_util::Compiler {
     output_file_name_ = file;
   }
 
-  // Returns false if any options are incompatible.  The num_files parameter
+  // Returns false if any options are incompatible. The num_files parameter
   // represents the number of files that will be compiled.
   bool ValidateOptions(size_t num_files);
 
   // Outputs to std::cerr the number of warnings and errors if there are any.
   void OutputMessages();
 
-  // When any files are to be compiled, they are compiled individually and
-  // written to separate output files instead of linked together.
-  void SetIndividualCompilationMode();
+  // Sets the flag to indicate individual compilation mode. In this mode, all
+  // files are compiled individually and written to separate output files
+  // instead of linked together. This method also disables linking and sets the
+  // output file extension to ".spv". Disassembly mode and preprocessing only
+  // mode override this mode and flags.
+  void SetIndividualCompilationFlag();
+
+  // Sets the flag to indicate disassembly mode. In this mode, the compiler
+  // emits disassembled textual output, instead of outputting object files.
+  // This method also sets the output file extension to ".s" and disables
+  // linking. This mode overrides individual compilation mode, and preprocessing
```

Comment:
I think preprocessing-only mode is better, just need to update the libshaderc comments to keep consistency. I will do that.